packages/flutter_tools/test/general.shard/drive/drive_service_test.dart

```diff
@@ -366,6 +368,29 @@ void main() {
     );
     await driverService.stop();
   });
+
+  testWithoutContext('WebDriver error message includes link to documentation', () async {
+    const String link = 'https://flutter.dev/docs/testing/integration-tests#running-in-a-browser';
+    final DriverService driverService = WebDriverService(
+      dartSdkPath: 'dart',
+      processUtils: ProcessUtils(
+        processManager: FakeProcessManager.empty(),
+        logger: BufferLogger.test(),
+      ),
+    );
+
+    try {
```

Comment:
This is dangerous because if we accidentally change this to not throw a tool exit the test will still pass. Consider something like this:

```dart
expect(() => driverService.startTest(...), throwsToolExit(message: ...))
```

Reply:
I agree, that's exactly what I was looking for and couldn't find, thanks!